com.github.opentyrian.OpenTyrian.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: com.github.opentyrian.OpenTyrian
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --device=all
+  - --share=network
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Pulseaudio access
+  - --socket=pulseaudio
+  #SDL needs to be able to talk to UPower
+  - --system-talk-name=org.freedesktop.UPower
+  #Allow SDL to inhibit the screensaver
+  - --talk-name=org.freedesktop.ScreenSaver
```

Comment:
I find these two weird. No other games need it, why OpenTyrian suddenly does?

Reply:
I've removed it